src/tool_configuration.rs

```diff
@@ -29,6 +29,20 @@ pub enum SkipExisting {
     All,
 }
 
+/// Container for the CLI test strategy
+#[derive(Debug, Clone, Copy, ValueEnum, Default)]
+pub enum TestStrategy {
+    /// Skip the tests
+    Skip,
+    /// Run the tests only if the build platform is the same as the host platform.
+    /// Otherwise, skip the tests. If the target platform is noarch,
+    /// the tests are always executed.
+    #[default]
```

Comment:
I am still undecided wether this or the `native-and-emulated` should be the default ...

Reply:
No real opinion on this. Will put `native-and-emulated`.